src/ui/loading-label.cpp

```diff
@@ -0,0 +1,36 @@
+#include "loading-label.h"
+
+namespace {
+
+const char *kRefreshIconFileName = ":/images/toolbar/refresh-alpha.png";
+const char *kOrangeIconFileName = ":/images/toolbar/refresh-orange-alpha@3x.png";
+
+} // namespace
+
+
+
+LoadingLabel::LoadingLabel(QWidget *parent)
```

Comment:
This class is unnecessary. Just use a QLabel + css can do it.

```css
QLabel#mLoadingLabel {
    background-image: url(:/images/toolbar/refresh.png);
}

QLabel#mLoadingLabel:hover {
    background-image: url(":/images/toolbar/refresh-orange.png");
}
```

The refresh signal can be implemented with an event filter in accout-view.cpp.